sim/midas/src/main/scala/midas/Config.scala

```diff
@@ -19,9 +19,23 @@ case object Platform extends Field[(Parameters) => PlatformShim]
 case object SynthAsserts extends Field[Boolean]
 case object SynthPrints extends Field[Boolean]
 
-// When False FpgaDebug() annotations are ignored
+/** When set, [[targetutils.FPGADebugAnnotation]]s (signals labelled with FPGADebug())
+  * are automatically wired out to an ILA.
+  */
 case object EnableAutoILA extends Field[Boolean](false)
 
+/** Sets a per-probe buffer depth on the ILA. A greater value permits capturing a longer
+  * waveform, at the expense of FPGA reasources. See PG172
+  * ([200~https://docs.xilinx.com/v/u/en-US/pg172-ila) for more info.
+  */
+case object ILADepthKey extends Field[Int](1024)
```

Comment:
Seems like this key is a duplicate of this key: https://github.com/firesim/firesim/blob/716b28d65438f94b1910c1ad658f5934fce75033/sim/firesim-lib/src/main/scala/util/Configs.scala#L16
https://github.com/firesim/firesim/blob/716b28d65438f94b1910c1ad658f5934fce75033/sim/firesim-lib/src/main/scala/util/Configs.scala#L9

Have you verified that this API still works?

Reply:
Good catch. I've fixed this.